exercises/leap/LeapSpec.groovy

```diff
@@ -0,0 +1,25 @@
+@Grab('org.spockframework:spock-core:1.0-groovy-2.4')
+import spock.lang.*
+
+class LeapSpec extends Specification {
+
+    def 'a year not divisible by 4 is not a leap year'() {
+        expect: new Year(2015).isLeapYear() == false
```

Comment:
What does the failure look like? Does the error message hint towards what to do next to fix it?